core/src/main/resources/jenkins/security/whitelisted-classes.txt

```diff
@@ -0,0 +1,149 @@
+com.google.common.collect.AbstractSetMultimap
+com.google.common.collect.EmptyImmutableList
+com.google.common.collect.EmptyImmutableSet
+com.google.common.collect.EmptyImmutableSortedSet
+com.google.common.collect.HashMultimap
+com.google.common.collect.ImmutableList
+com.google.common.collect.ImmutableSet$SerializedForm
+com.google.common.collect.ImmutableSortedSet
+com.google.common.collect.RegularImmutableList
+com.google.common.collect.RegularImmutableSet
+com.google.common.collect.RegularImmutableSortedSet
+com.google.common.collect.SingletonImmutableList
+com.google.common.collect.SingletonImmutableMap
+com.google.common.collect.SingletonImmutableSet
+
+# TODO remove when maven-plugin 3.1 is widely adopted
+hudson.maven.MavenInformation
+
+# TODO remove when https://github.com/jenkinsci/kubernetes-pipeline-plugin/pull/66 is widely adopted
+io.fabric8.docker.api.model.Config
+io.fabric8.docker.api.model.GraphDriverData
+io.fabric8.docker.api.model.ImageInspect
+
+java.io.File
+java.lang.Boolean
+java.lang.Byte
+java.lang.Character
+java.lang.Class
+java.lang.Double
+java.lang.Enum
+java.lang.Float
+java.lang.Integer
+java.lang.Long
+java.lang.Number
+java.lang.Object
+java.lang.Short
+java.lang.StackTraceElement
+java.lang.String
+java.lang.reflect.Proxy
+java.net.URI
+java.net.URL
+java.security.KeyRep
```

Comment:
@jglick Do you remember what the use case for commit e8768491f62addc7fd1966db4b4e194ba186e82a was? I'm wondering if we need to be worried about XStream trying to serialize/deserialize these types on Java 17, where it would probably fail because we haven't opened those modules.

Reply:
> Do you remember what the use case for commit e876849 was?

Sorry, no. Probably there was a test failure somewhere, in which case there is no need to worry (it would show up if this is really a problem on 17).

> XStream trying to serialize/deserialize these types

More likely in Remoting.